core/simpleproducer.go

```diff
@@ -123,6 +125,19 @@ func (prod *SimpleProducer) Configure(conf PluginConfigReader) error {
 	dropStreamID := StreamRegistry.GetStreamID(conf.GetString("DropToStream", DroppedStream))
 	prod.dropStream = StreamRegistry.GetRouterOrFallback(dropStreamID)
 
+	// Simple health check for the plugin state
+	//   Path: "/<plugin_class>/<plugin_id>/SimpleProducer/pluginstate"
```

Comment:
As we already discussed in the office - the question is we really need multiple health checks in a plugin or not.